node/src/actors/sessions_manager/mod.rs

```diff
@@ -129,13 +130,18 @@ impl SessionsManager {
 
     /// Method to periodically discover peers
     fn discovery_peers(&self, ctx: &mut Context<Self>, discovery_peers_period: Duration) {
+        let peers_manager_addr = PeersManager::from_registry();
         // Schedule the discovery_peers with a given period
         ctx.run_later(discovery_peers_period, move |act, ctx| {
             // Send Anycast(GetPeers) message
             ctx.notify(Anycast {
                 command: SendGetPeers {},
                 safu: false,
             });
+
+            // Send MeltPeers message
```

Comment:
Explain what it does and why do it here:

```suggestion
            // Remove peers from ice bucket that have been "melted"
            // This peers will be added to new again, to ensure that they never get stuck in the ice bucket
```